public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -297,9 +336,15 @@ export function FolderPicker(props: Props) {
   }
 }
 
-function mapSearchHitsToOptions(hits: DashboardSearchHit[], filter?: FolderPickerFilter) {
+function mapSearchHitsToOptions(hits: DashboardSearchHit[], filterSlashes: boolean, filter?: FolderPickerFilter) {
   const filteredHits = filter ? filter(hits) : hits;
-  return filteredHits.map((hit) => ({ label: hit.title, value: hit.id }));
```

Comment:
I think we should be able to move this filter to the `useRuleFolderFilter` hook. It's an alerting-only filtering hook